### ISSUE TITLE
Stuck at tty, fails deployment -- This account is currently not available.

### ISSUE THREAD

**Issue by Kreyren** [nix-community/nixos-anywhere#357]
TL;DR: Minimal aarch64-linux configuration fails to deploy via nixos-anywhere.

Using commit [NiXium-org/NiXium@`833c894` (#118)](https://github.com/NiXium-org/NiXium/pull/118/commits/833c894ca8f741e5c6b9f920ce1974a2cfefb736) and trying to deploy configuration `nixos-tsvetan-stable` from  https://github.com/NiXium-org/NiXium/pull/118/files#diff-b86556436e0d947662a255af625277f18d3bce762b61122855939e2e0fdf3a95R9 via custom aarch64-sd image https://github.com/NiXium-org/NiXium/pull/118/files#diff-ca60330c0106d573414c9e1834fea82c841d3db25aaa3d5598460f9db14b942dR9 to get over the lack of space on the device and using novel task called 'hijack' https://github.com/NiXium-org/NiXium/pull/118/files#diff-8c3a3c8af3cb0507d5f7e72af04689c687aab023f149f151015e41377d8a60a2 causes the image to be stuck:

![image](https://github.com/user-attachments/assets/d438d708-2cd1-4519-8493-21e52f9af73d)

Same issue observed while trying to use nixos-anywhere on a x86_64 qemu guest.

Log from nixos-anywhere: https://gist.githubusercontent.com/Kreyren/4420dc22d7d9e5688a19e4d3a52f3745/raw/9662b72121ce17c38d9655cf51c7fb8de122d2f4/gistfile1.txt

Log from serial console: https://gist.githubusercontent.com/Kreyren/fcc44ca0ba258cbd3beedf64c09d0259/raw/d92f326dd7e14f8e8fd285c155716552413e9413/gistfile1.txt

Deploying the same configuration via `disko-install` appears to have no issues.

**Comment by sedlund** [comment]
what is stuck?

it looks like you logged in with root after install, no?

**Comment by Mic92** [comment]
I believe this was fixed in https://github.com/nix-community/nixos-images/pull/272/commits/b81c6e0ace8c3109352be28bb9ed9a856a215ea2
